pi-hole.json

```diff
@@ -0,0 +1,83 @@
+{
+    "Pi-Hole": {
+        "containers": {
+            "pi-hole-diginc": {
+                "image": "diginc/pi-hole",
+                "opts": [
+                    [
+                        "--cap-add",
+                        "NET_ADMIN"
+                    ],
+                    [
+                	"--restart",
+                	"always"
+            	    ]
+                ],
+                "launch_order": 1,
+                "ports": {
+                    "53": {
+                        "description": "DNS port. Required: 53",
+                        "host_default": 53,
+                        "label": "DNS-Port",
+                        "ui": false
+                    },
+                    "80": {
+                        "description": "Web-port. Recommended: 80",
+                        "host_default": 80,
+                        "label": "Web-Port",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/etc/pihole": {
+                        "description": "Choose a Share for Plex Requests configuration. Eg: create a Share called pihole-config for this purpose alone.",
```

Comment:
Replace old reference to Plex Requests with something related to Pi-hole, such as "Choose a share for Pi-Hole configuration files", for instance.